src/components/pages/Room/Rooms.js

```diff
@@ -0,0 +1,93 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Carousel from 'nuka-carousel/lib/carousel';
+import { useTheme } from '@mui/material/styles';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import { getRooms } from '../../../redux/roomsSlice';
+import Room from './Room';
+import NoData from '../../common/NoData';
+import { isNilOrEmpty } from '../../../helpers/utils';
+import './Carousel.css';
+
+function Rooms() {
```

Comment:
- [ ] You are using the "function" keyword for some of your functions and components.  Please, refactor those functions to ES6 arrow functions.

## Explanation

All over the software industry, the "function" keyword has been soft deprecated in favor of ES6 functions.

## Tips and extra help

Just do a quick search in Vscode for the word "function" so you can find them all, and change each one to ES6 arrow functions.  Don't worry, it should take no more than 5-10 minutes.  

That way,  your app will comply with the latest standards 👍